.babelrc

```diff
@@ -9,5 +9,6 @@
       }
     ],
     "react"
-  ]
+  ],
+  "plugins": ["transform-object-rest-spread"]
```

Comment:
Noticed this plugin was missing to do the rest and spread operators in js files!